libp2p/NodeTable.h

```diff
@@ -269,7 +273,9 @@ class NodeTable : UDPSocketEvents
     std::unique_ptr<NodeTableEventHandler> m_nodeEventHandler;		///< Event handler for node events.
 
     NodeID const m_hostNodeID;
-    NodeIPEndpoint m_hostNodeEndpoint;
+
+    /// Host's endpoint if p2p is started, otherwise an endpoint specific to the NodeTable
+    NodeIPEndpoint m_nodeEndpoint;
```

Comment:
I think it was clearer as `m_hostNodeEndpoint`, less ambiguity about which node is that (as there are many nodes in `NodeTable` code)

> /// Host's endpoint if p2p is started, otherwise an endpoint specific to the NodeTable
What do you mean? it's the same thing, no?

Reply:
No, from what I understand the node table only shares the host's endpoint if p2p is running (i.e. the UDP socket is bound to the same IP/port as the host). If there's no p2p then the endpoint isn't shared with any other component and is only used by the node table.